src/WebJobs.Script/EnvironmentSettingNames.cs

```diff
@@ -15,10 +15,10 @@ public static class EnvironmentSettingNames
         public const string AzureWebsitePlaceholderMode = "WEBSITE_PLACEHOLDER_MODE";
         public const string AzureWebsiteHomePath = "HOME";
         public const string AzureWebJobsScriptRoot = "AzureWebJobsScriptRoot";
+        public const string AzureWebJobsEnvironment = "AzureWebJobsEnv";
```

Comment:
>AzureWebJobsEnvironment [](start = 28, length = 23)

What is this? (I wish some of these vars had a common about what their semantics are)

Reply:
/cc @mamaso

Reply:
That is actually dead code, will remove. Was used to enable debugging for the node language worker (https://github.com/Azure/azure-webjobs-sdk-script/pull/1811)